### PR TITLE
elastic: init at 0.1.3

### PR DESCRIPTION
**Pull request by 0xMRTT**
###### Description of changes

Elastic allows to design and export spring physics-based animations to use with libadwaita and to export them to C, JavaScript, Python, Vala or Rust code.

See https://apps.gnome.org/en/app/app.drey.Elastic/

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -100,6 +100,13 @@
     name = "Daniel Glinka";
     github = "0xd61";
     githubId = 8351869;
+  _0xMRTT = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:103]:
```suggestion
  };
  _0xMRTT = {
```

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -100,6 +100,13 @@
...
+    email = "0xMRTT@proton.me";
+    name = "0xMRTT";
+    gitthub = "0xMRTT";
+    githubId = 105598867;
+    matrix = "@0xmrtt:envs.net";
+  };
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:109]:
```suggestion
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please follow the [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes) when naming your commits.

File: pkgs/applications/misc/elastic/default.nix [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    libgee
+    gtksourceview5
+    template-glib
+  ];
+
+  meta = with lib; {
+    description = "Design spring animations";
+    homepage = "https://gitlab.gnome.org/World/elastic/";
+    mainProgram = "app.drey.Elastic";
+    license = licenses.gpl3;
```
Comment (jtojnar) [inline pkgs/applications/misc/elastic/default.nix:54]: `gpl3` is unclear, use `gpl3Plus` or `gpl3Only`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

**Comment by 0xMRTT** [comment]
@SuperSandro2000 Is there something that needs to be improved or can this PR be merged ?

**Pull request merged**